modin/test/storage_formats/pandas/test_internals.py

```diff
@@ -901,3 +901,42 @@ def test_shuffle_partitions_with_empty_pivots(ascending):
 
     assert new_partitions.shape == (1, 1)
     assert ref.equals(res)
+
+
+@pytest.mark.parametrize("has_cols_metadata", [True, False])
+@pytest.mark.parametrize("has_dtypes_metadata", [True, False])
+def test_merge_preserves_metadata(has_cols_metadata, has_dtypes_metadata):
```

Comment:
same

Reply:
This is just testing cache (whether it's preserved or not), there's no need to make this test multi-partitioned